extern/cloop/src/cloop/cloop.vcxproj

```diff
@@ -20,8 +20,14 @@
   </ItemGroup>
   <PropertyGroup Label="Globals">
     <Keyword>Win32Proj</Keyword>
+    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
+    <PlatformToolset>v141</PlatformToolset>
```

Comment:
This change broke VS 2015 build.

Reply:
Probably small makefile to build CLOOP would be best to fix it.

Reply:
> 
> 
> This change broke VS 2015 build.

Fortunately official compiler for Firebird 4 is VS 2017 so 2015 can be simply dropped.